### PR TITLE
UPDATE: Fix Sign Up issue + simple time reminder implemented

### PR DESCRIPTION
**Pull request by tjw0612**
## Why

This pull request is needed because:

Implemented a simple counter in the responses form to act as a reminder
image

## What

This pull request introduces the following:

 * `Changes introduced`

`Add Implementation details if needed`

### Screenshot

`Attach any screenshots here. Remove when unneeded`

## How to Test

`Give the reviewer(s) some clues on how to test your PR`

## Other

`This section is for things that can't be put into the previous sections. Remove when unneeded`

### CODE REVIEW

File: app/views/chomp_sessions/show.html.erb [hunk 1]
```diff
@@ -2,10 +2,12 @@
   <div class="row justify-content-center">
     <div class="col-12 col-lg-8">
       <div class="form-login">
-        <h1 class="mb-3">Input Preferences</h1>
+        <h1 class="mb-3">Input Preferences</h1> <%= render "timer" %>
```
Comment (mxng) [inline app/views/chomp_sessions/show.html.erb]: can move the <render timer> into the next line?

**Comment by mxng** [comment]
Screenshot 2021-09-28 at 3 14 45 PM

File: app/views/chomp_sessions/show.html.erb (hunk 1 above)
Comment (tjw0612) [reply app/views/chomp_sessions/show.html.erb]: render moved to next line in latest push

**Comment by tjw0612** [comment]
image

**Review by TeeWhyKay — APPROVED** [review]
looks fine, hopefully registration works

**Pull request merged**